learn/framework/overview.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Overview
+description: Substrate is a powerful blockchain development framework that provides many ready-to-use functionalities allowing product teams to quickly and easily build Appchains.
+---
+
+# Overview {: #overview } 
+
+## Building an Appchain from Scratch {: #building-an-appchain-from-scratch } 
+
+Building an Appchain from scratch is a very complex task that requires deep knowledge in a wide range of areas, including (but not limited to):
+
+- **Consensus Algorithms** - Consensus ensures that all participants in the blockchain network agree on the validity of transactions. Some popular consensus mechanisms include Proof of Work (PoW) and Proof of Stake (PoS)
+
+- **Cryptography** - Cryptography plays a crucial role in securing the blockchain. You'll need cryptographic algorithms for tasks like creating digital signatures, verifying transactions, and encrypting data
+
+- **Distributed Network** - A network architecture to enable nodes to communicate, validate transactions, and synchronize the blockchain data is key to maintaining a shared ledger in a decentralized network
+
+- **Data Structures** - Besides the list of blocks, where each block contains a set of transactions along with a reference to the previous block, an optimized and performant strategy to store the state of the network is needed
+
+- **Governance** - Define how decisions are made, how upgrades or modifications are implemented, and how conflicts are resolved within the network
+
+Fortunately, there is no need to write implementations of these blockchain components for there is an awesome open-source framework called [Substrate](https://substrate.io/){target=_blank}.
+
+## Substrate Framework {: #substrate-framework}
```

Comment:
I think we should do a version of -> https://d33wubrfki0l68.cloudfront.net/35f51744b3527f05d05b415dd54c74c666fe2eb3/161b1/static/64b2fcb61748ae77f4dd4c9ce63872b1/ee8ba/compose-runtime.png

To explain that Substrate has a bunch of pre-made modules and you can use the subset that you think is best.

Also, images are always better than just a bunch of text

Reply:
Added!